bundler/lib/bundler/definition.rb

```diff
@@ -528,6 +536,23 @@ def precompute_source_requirements_for_indirect_dependencies?
       @remote && sources.non_global_rubygems_sources.all?(&:dependency_api_available?) && !sources.aggregate_global_source?
     end
 
+    def pin_locally_available_names(source_requirements)
+      new_source_requirements = {}
```

Comment:
🤔 you can use something like `{}.tap do |new_source_requirements|` to prevent explicit return value at the end of method

Reply:
Indeed, I will update it as you suggest 👍.

Reply:
I changed it to use `each_with_object`, what do you think?